src/js/index.js

```diff
@@ -32,6 +38,17 @@ cards.forEach(card => {
   })
 })
 
+forms.forEach(form => {
+  const textarea = form.querySelector('[data-js="textarea"]')
+  const counter = form.querySelector('[data-js="counter"]')
+
+  textarea.addEventListener('input', () => {
+    const textLength = textarea.value.length
+    counter.innerHTML =
+      textarea.maxLength - textLength + '/' + `${textarea.maxLength}`
```

Comment:
You could simplify that last String. No template string needed, if you just want to render the same String there.